### ISSUE TITLE
Add way to view topic list from inside thread narrow

### ISSUE THREAD

**Issue by WesleyAC** [zulip/zulip-mobile#4529]
If a user wants to get to the "topic list" view in a particular stream, they have no way of doing that that does not involve marking some messages as read. This is OK in the case when they want to look at a thread that they have unreads in, since they can view the thread from the home screen, but in the case where they're trying to get to an old thread, it can be frustrating.

A incomplete, but useful solution to this would be to add a copy of the "thread list" hamburger icon to the top bar when the user is in a narrow, next to the un-narrow/up arrow icon.

**Comment by gnprice** [comment]
Incomplete because you still have to first get to some topic narrow in the same stream, right? But I agree this would be a helpful step.

Adding the thread list icon to the end of the app bar would be reasonable; the main downside is that three icons is kind of a lot of them to have there.

A variation on that would be if we add an overflow menu, as described at https://github.com/zulip/zulip-mobile/issues/4532#issuecomment-799152746 -- then I think this would be a natural action to add to that menu too.

**Comment by WesleyAC** [comment]
> Incomplete because you still have to first get to some topic narrow in the same stream, right? But I agree this would be a helpful step.

Yep. Perhaps clicking on the stream name in the main view should bring up the topic list instead of all messages view, or something like that. But that seems more likely to frustrate people (as we've seen with Recent Topics on the webapp) and not super high priority.

I plan on implementing this at the same time as #4532, so let's have discussion on the UI for this in that issue.
